packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.js

```diff
@@ -0,0 +1,230 @@
+import SfIcon from "@/components/atoms/SfIcon/SfIcon.vue";
+import SfLoader from "@/components/atoms/SfLoader/SfLoader.vue";
```

Comment:
don't use webpack aliases on components, it will break the imports in the app because they are not present there ;)

Reply:
Done